src/components/toast/Toast.jsx

```diff
@@ -0,0 +1,26 @@
+import { IonToast, CreateAnimation, Animation } from "@ionic/react";
+
+/**
+ * React Wrapper for IonToast
+ */
+const Toast = ({ showToast, setShowToast, toastMessage, dismissBtnHandler, color }) => {
+  const dismissBtn = {
+    text: "Okay",
+    role: "cancel",
+    handler: dismissBtnHandler,
+  };
+
```

Comment:
It might be better to just use like a close ("X") button at the topleft or topright of the toast